src/utils/streams/replace_stream.test.js

```diff
@@ -36,66 +34,68 @@ async function concatToString(streams) {
 }
 
 describe('replaceStream', () => {
-  it('produces buffers when it receives buffers', async () => {
+  test('produces buffers when it receives buffers', async () => {
     const chunks = await createPromiseFromStreams([
       createListStream([Buffer.from('foo'), Buffer.from('bar')]),
       createReplaceStream('o', '0'),
       createConcatStream([])
     ]);
 
     chunks.forEach(chunk => {
-      expect(chunk).to.be.a(Buffer);
+      expect(chunk).toBeInstanceOf(Buffer);
     });
   });
 
-  it('produces buffers when it receives strings', async () => {
+  test('produces buffers when it receives strings', async () => {
     const chunks = await createPromiseFromStreams([
       createListStream(['foo', 'bar']),
       createReplaceStream('o', '0'),
       createConcatStream([])
     ]);
 
     chunks.forEach(chunk => {
-      expect(chunk).to.be.a(Buffer);
+      expect(chunk).toBeInstanceOf(Buffer);
     });
   });
 
-  it('expects toReplace to be a string', () => {
-    expect(() => createReplaceStream(Buffer.from('foo')))
-      .to.throwError(error => {
-        expect(error.message).to.match(/be a string/);
-      });
+  test('expects toReplace to be a string', () => {
+    try {
+      createReplaceStream(Buffer.from('foo'));
```

Comment:
`expect(() => createReplaceStream(Buffer.from('foo'))).toThrowError(/be a string/)` matches the original test a little better